examples/portfolio/src/components/MainHead.astro

```diff
@@ -1,8 +1,8 @@
 ---
-import '../styles/global.scss';
+import "../styles/global.css";
```

Comment:
Noticing single quotes were changed to double quotes in JS blocks. Sounds like this _shouldn't_ be our monorepo default. Flagging to change

Reply:
ran prettier!